### PR TITLE
Add putEnvInState

### PR DESCRIPTION
**Pull request by Dawidpol**

### CODE REVIEW

**Comment by yujinlim** [comment]
@Dawidpol you might need to update webpack to compile to node target instead of UMD.

**Comment by yujinlim** [comment]
@Dawidpol process is a global variable, thus, you dont have to include that as part of your requiring module.

File: src/manipulation.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import * as process from 'process'
```
Comment (matiboy) [inline src/manipulation.ts:1]: @Dawidpol That's your issue I would say. Do not import process

Comment (yujinlim) [reply src/manipulation.ts:1]: @matiboy i thought of that too, but is not.

**Comment by yujinlim** [comment]
@Dawidpol is able to access process.env now. is this what you are expecting?

**Comment by Dawidpol** [comment]
@yujinlim When i changed target to node it is working

**Comment by yujinlim** [comment]
@matiboy https://travis-ci.org/redapesolutions/koalambda/jobs/339286504 can avoid error on this commit, https://github.com/redapesolutions/koalambda/pull/5/commits/6991aa749087fc88a60e72947400f58ce0d3d8ca. This is throwing error is because we are requiring to build the lib folder before we push. In the case that we did not push it, test command will use the old lib files and fails

**Comment by yujinlim** [comment]
@Dawidpol then, change the PR title to more happier mode. 😆

**Comment by matiboy** [comment]
@yujinlim What do you suggest on the Travis CI job error you mentioned?

**Pull request merged**